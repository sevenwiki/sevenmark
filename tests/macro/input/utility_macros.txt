Before null macro [null] after null macro.