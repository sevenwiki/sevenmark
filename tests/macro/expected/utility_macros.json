[
  {
    "Text": {
      "location": {
        "start": 0,
        "end": 18
      },
      "content": "Before null macro "
    }
  },
  "Null",
  {
    "Text": {
      "location": {
        "start": 24,
        "end": 42
      },
      "content": " after null macro."
    }
  }
]